spec/lib/reports/bulk_coop_report_spec.rb

```diff
@@ -191,6 +191,48 @@ module BulkCoop
           end
         end
       end
+
+      describe SupplierReport do
+        let(:d1) { create(:distributor_enterprise) }
+        let(:oc1) { create(:simple_order_cycle) }
+        let(:o1) { create(:order, completed_at: 1.day.ago, order_cycle: oc1, distributor: d1) }
+        let(:li1) { build(:line_item_with_shipment) }
+        let(:li2) { li1.dup }
+        let(:li3) { build(:line_item_with_shipment) }
+        let(:params) { { display_summary_row: true } }
+        let(:user) { create(:admin_user) }
+        let(:report){ SupplierReport.new user, params }
```

Comment:
As the `params` are small, I think it would be appropriate to simply include them inline here. This makes it easier to read.
In fact, event the `create(:admin_user)` could be included here, considering that we don't need to refer to the variable anywhere else.